Source/AudibleUtilities/AccountsSettings.cs

```diff
@@ -47,6 +47,22 @@ private List<Account> _accounts_json
 				update_no_validate();
 			}
 		}
+
+		private string _cdm;
+		[JsonProperty]
+		public string Cdm
+		{
+			get => _cdm;
+			set
+			{
+				if (value is null)
+					return;
+
+				_cdm = value;
+				update_no_validate();
+			}
+		}
```

Comment:
Store the CDM file as a base64  string in the AccountSetting.json file. A CDM isn't linked with a specific account, so a user only need one CDM for all their accounts.